examples/language-modeling/requirements.txt

```diff
@@ -5,3 +5,4 @@ protobuf
 evaluate
 scikit-learn
 peft == 0.6.2
+pytest == 7.4.2
```

Comment:
which use case need this fix?

Reply:
examples/language-modeling/run_clm.py
when this example is run with pytest >= 7.4.2 it breaks due to hardcoded path in transformers/testing_utils.py", line 129